src/eu/packages/eu-component-link/docs/link.md

```diff
@@ -0,0 +1,42 @@
+# ECL Twig - EU Link component
+
+npm package: `@ecl-twig/eu-component-link`
+
+```shell
+npm install --save @ecl-twig/eu-component-link
+```
+
+## Link
+
+### Parameters
+
+- "link" (associative array) (default: 'predefined structure below')
+  - "type" (string) (default: '') - type of link. Available types are 'default' or standalone
+  - "label" (string) (default: '') - Content of link
+  - "path" (string) (default: '') - Link url (href attribute)
+  - "icon_position" (string) (default: 'after') - Position of link icon (can be 'before' or 'after') if icon is available
+- "icon" [optional] (associative array) default: A predefined structure such as in the Icon component. All parameters can be freely set, except for the 'size' parameter, which is set permanently as 'fluid'.
+- "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the icon
+- "extra_attributes" (optional) (array) (default: []) Extra attributes for icon
+  - "name" (string) Attribute name, eg. 'data-test'
+  - "value" (string) Attribute value, eg: 'data-test-1'
+
+### Example:
+
```

Comment:
Please add `<!-- prettier-ignore -->` to prevent extra formatting to be removed (see discussion on slack)

Reply:
Fixed in https://github.com/ec-europa/ecl-twig/pull/31/commits/45a6305895c369887c37648d7ec8c93b36c7d32b